binaries/bzutil/main.go

```diff
@@ -43,6 +43,7 @@ var supportedCommands map[string]bool = map[string]bool{
 
 func main() {
 	log.AddHook(hooks.NewContextHook())
+	log.SetLevel(log.InfoLevel)
```

Comment:
I think this is the default... Could take a log_level flag if we want to be able to enable debug level similar to the other ./binaries/...

Reply:
Default seemed to be error which is why I added - logs had disappeared! (adding level flags)